### PR TITLE
#2047 - Change BC Sans font with version 2

### PR DESCRIPTION
**Pull request by sh16011993**
As a part of this PR, upgraded the BC Sans from version 1 to version 2. This upgrade allows for the support of indigenous languages.

The following symbols are tested:

image

Screenshots related to testing:

Vuetify testing (Veutify --> database --> webpage) :

Database - 
image

Webpage -
image

Formio testing (Formio --> database --> webpage) :

Formio -
image

Database - 
image

Webpage -
image

### CODE REVIEW

**Review by ann-aot — COMMENTED** [review]
Just wanted to know why 5 files are and only 3 files are added. What about other files?

**Comment by ann-aot** [comment]
I remember the link shared by @hellolynn-tbtb  to download the new files was not working, would like to know, from where did u get these new files?

**Comment by ann-aot** [comment]
> Just wanted to know why 5 files are and only 3 files are added. What about other files?

we have 8 files, right? 
image

**Comment by sh16011993** [comment]
> I remember the link shared by @hellolynn-tbtb to download the new files was not working, would like to know, from where did u get these new files?

@andrewsignori-aot shared the [BC Sans](https://github.com/bcgov/bc-sans/tree/master/fonts) github project link.

**Comment by sh16011993** [comment]
> > Just wanted to know why 5 files are and only 3 files are added. What about other files?
> 
> we have 8 files, right? image

All the 8 files are added.

**Comment by sh16011993** [comment]
> Just wanted to know why 5 files are and only 3 files are added. What about other files?

There are 8 files.

**Comment by ann-aot** [comment]
> > I remember the link shared by @hellolynn-tbtb to download the new files was not working, would like to know, from where did u get these new files?
> 
> @andrewsignori-aot shared the [BC Sans](https://github.com/bcgov/bc-sans/tree/master/fonts) github project link.

Thanks for sharing the link, a question for the dev @andrewsignori-aot , @dheepak-aot , @guru-aot, and @hellolynn-tbtb , I see there are 12 files if have only 8 files in our project. shouldn't we add all 12 files?

**Comment by dheepak-aot** [comment]
> > > I remember the link shared by @hellolynn-tbtb to download the new files was not working, would like to know, from where did u get these new files?
> > 
> > 
> > @andrewsignori-aot shared the [BC Sans](https://github.com/bcgov/bc-sans/tree/master/fonts) github project link.
> 
> Thanks for sharing the link, a question for the dev @andrewsignori-aot , @dheepak-aot , @guru-aot, and @hellolynn-tbtb , I see there are 12 files https://github.com/bcgov/bc-sans/tree/master/fonts, but we have only 8 files in our project. shouldn't we add all 12 files?

Hello team, when I clicked at this link, I was able to download all the version2 of BC Sans Font. 
image

It has both woff and woff2 font files

image

**Comment by sh16011993** [comment]
> > > > I remember the link shared by @hellolynn-tbtb to download the new files was not working, would like to know, from where did u get these new files?
> > > 
> > > 
> > > @andrewsignori-aot shared the [BC Sans](https://github.com/bcgov/bc-sans/tree/master/fonts) github project link.
> > 
> > 
> > Thanks for sharing the link, a question for the dev @andrewsignori-aot , @dheepak-aot , @guru-aot, and @hellolynn-tbtb , I see there are 12 files https://github.com/bcgov/bc-sans/tree/master/fonts, but we have only 8 files in our project. shouldn't we add all 12 files?
> 
> Hello team, when I clicked at this link, I was able to download all the version2 of BC Sans Font. image
> 
> It has both woff and woff2 font files
> 
> image
> 
> I believe the github link must be considered as source to download fonts. But not sure.
> 
> @ann-aot Thanks for bringing the point. IMO yes we must add the remaining fonts as well.

I had called the BC Service help when I wasn't able to download the files and they must have fixed it.

File: sources/packages/web/src/assets/fonts/BCSans/BCSans-Bold.woff
Comment (andrewsignori-aot) [inline sources/packages/web/src/assets/fonts/BCSans/BCSans-Bold.woff:1]: Should we update our `BCSans.scss` as below and just ensure that we have those files associated?
https://developer.gov.bc.ca/Typography#font-face

Comment (guru-aot) [reply sources/packages/web/src/assets/fonts/BCSans/BCSans-Bold.woff:1]: I believe that should be enought too.

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for looking into the npm package, looks good 👍

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @sh16011993

**Review by ann-aot — APPROVED** [review]
Thanks for doing the changes 👍 LGTM

**Review by dheepak-aot — APPROVED** [review]
Great job!

**Pull request merged**